horace_core/sqw/PixelData/@PixelData/private/check_pixel_fields_.m

```diff
@@ -1,21 +1,29 @@
-function check_pixel_fields_(obj, fields)
+function indexes = check_pixel_fields_(obj, fields)
```

Comment:
Are we using `indexes` or `indices` elsewhere?

Reply:
If Harry decided to use American spelling here -- I do not mind (I have extracted out what was inside)